lib/sport_ngin_aws_auditor/instance_helper.rb

```diff
@@ -144,6 +163,21 @@ def filter_ris_region_based(ris)
       ris.select { |ri| ri.scope == 'Region' }
     end
 
+    # this breaks up the instances array into instances with 'auto' or 'kitchen' in the name and instances without
+    def filter_ignored_instances(instances)
+      instances.partition { |instance| instance.name.include?("auto") || instance.name.include?("kitchen") }
```

Comment:
The list of ignored name patterns should be configurable, not hard coded into the open source gem.